user-management/src/main/elm/sources/JsonEncoder.elm

```diff
@@ -20,11 +20,12 @@ encodeUser (user, password) =
     , ("role", list (\s -> string s) (user.authz ++  user.role))
     ]
 
-encodeAddUser: (User, String, Bool) -> Value
-encodeAddUser (user, password, isPreHashed) =
+encodeAddUser: User -> String -> Bool -> Bool -> Value
```

Comment:
I think you should have a type to regroupe the 3/4 values  (name it "AddUserForm" or something likewise